### PR TITLE
b value selection fix and test, power map test, models moved to own submodule

### PR DESCRIPTION
**Pull request by 36000**
I found a bug in the b value selection code that is now fixed and tested for.

### CODE REVIEW

File: AFQ/tests/test_api.py [hunk 1]
```diff
@@ -168,6 +168,25 @@ def test_AFQ_data():
         myafq.export_rois()
 
 
+def test_AFQ_anisotropic():
+    """
+    Test if API can run without prealign
```
Comment (arokem) [inline AFQ/tests/test_api.py:173]: Docstring needs an update

File: AFQ/tests/test_api.py [hunk 2]
```diff
@@ -168,6 +168,25 @@ def test_AFQ_data():
...
+    myafq = api.AFQ(
+        bids_path=bids_path,
+        dmriprep='vistasoft',
+        segmentation='freesurfer',
+        min_bval=1990,
+        max_bval=2010,
+        b0_threshold=50,
+        reg_template="mni_T1",
+        reg_subject="power_map")
+    myafq.export_rois()
```
Comment (arokem) [inline AFQ/tests/test_api.py:187]: Can we do something more specific here to check that the selection works as expected? I guess we can call individual (private) methods of this object once it’s initialized.

Comment (arokem) [inline AFQ/tests/test_api.py:187]: Can we do something more specific here to check that the selection works as expected? I guess we can call individual (private) methods of this object once it’s initialized.

Comment (36000) [reply AFQ/tests/test_api.py:203]: Is this similar to what you had in mind?

Comment (arokem) [reply AFQ/tests/test_api.py:203]: Can we maybe check that the APM is where we'd expect it to be at this point?

Comment (36000) [reply AFQ/tests/test_api.py]: I am now checking that the APM map is roughly similar (rounded to the ten's place) to the map in file_dict. Is there a better way to check the mapping?

**Comment by 36000** [comment]
Our tests officially take longer than an hour now!  :tada:

**Comment by 36000** [comment]
move apm calculations into apm.py

**Comment by arokem** [comment]
Great. I think this is now ready to merge. I'll just wait for the CI to finish and then go ahead.

**Pull request merged**